### PR TITLE
SONAR-9508 Create api/ce/worker_count WS

### PR DESCRIPTION
**Pull request by julienlancelot**

### CODE REVIEW

File: server/sonar-server/src/test/java/org/sonar/server/ce/ws/CeWsModuleTest.java [hunk 1]
```diff
@@ -30,6 +30,6 @@
   public void verify_count_of_added_components() {
     ComponentContainer container = new ComponentContainer();
     new CeWsModule().configure(container);
-    assertThat(container.size()).isEqualTo(11 + 2 /* injected by ComponentContainer */);
+    assertThat(container.size()).isEqualTo(12 + 2 /* injected by ComponentContainer */);
```
Comment (sns-seb) [inline server/sonar-server/src/test/java/org/sonar/server/ce/ws/CeWsModuleTest.java]: FYI, `2` can be replaced by `ComponentContainer.COMPONENTS_IN_EMPTY_COMPONENT_CONTAINER`

File: server/sonar-server/src/main/java/org/sonar/server/ce/ws/WorkerCountAction.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import org.sonarqube.ws.WsCe.WorkerCountResponse;
+
+import static org.sonar.server.ws.WsUtils.writeProtobuf;
+
+public class WorkerCountAction implements CeWsAction {
+
+  public static final String ACTION = "worker_count";
+
+  private final UserSession userSession;
+  private final WorkerCountProvider workerCountProvider;
```
Comment (sns-seb) [inline server/sonar-server/src/main/java/org/sonar/server/ce/ws/WorkerCountAction.java]: missing `@CheckForNull`  to benefit from SQ's NPE check rules?

File: server/sonar-server/src/main/java/org/sonar/server/ce/ws/WorkerCountAction.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  public void handle(Request wsRequest, Response wsResponse) throws Exception {
+    userSession.checkIsSystemAdministrator();
+    writeProtobuf(createResponse(), wsRequest, wsResponse);
+  }
+
+  private WorkerCountResponse createResponse(){
+    WorkerCountResponse.Builder builder = WorkerCountResponse.newBuilder();
+    if (workerCountProvider == null) {
+      return builder
+        .setValue(1)
```
Comment (sns-seb) [inline server/sonar-server/src/main/java/org/sonar/server/ce/ws/WorkerCountAction.java]: magic number?

File: server/sonar-server/src/main/java/org/sonar/server/ce/ws/WorkerCountAction.java (hunk 2 above)
Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/ce/ws/WorkerCountAction.java]: I'm not sure it's working when used on variable ?

File: server/sonar-server/src/main/java/org/sonar/server/ce/ws/WorkerCountAction.java (hunk 3 above)
Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/ce/ws/WorkerCountAction.java]: :) 
I'll replace it with a constant, you're right

**Pull request merged**